src/components/templates/AnimateMap/components/FirebarrelWidget/FirebarrelWidget.tsx

```diff
@@ -11,13 +11,13 @@ import { AnimateMapVenue } from "types/venues";
 import { useWorldUsersById } from "hooks/users";
 import { useUser } from "hooks/useUser";
 
-import LocalParticipant from "components/organisms/Room/LocalParticipant";
+import { LocalParticipant } from "components/organisms/Room/LocalParticipant";
 import VideoErrorModal from "components/organisms/Room/VideoErrorModal";
 
 import { Button } from "components/atoms/Button";
 
 import { WithId } from "../../../../../utils/id";
-import Participant from "../../../../organisms/Room/Participant";
+import { Participant } from "../../../../organisms/Room/Participant";
```

Comment:
please, make these imports absolute and sort them out. 🙃

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2221/commits/34c2da4fc00fb2f75d093deb88233a7c5846ec92